app/projectsmodel.h

```diff
@@ -88,7 +88,24 @@ class ProjectModel : public QAbstractListModel
 
       bool operator < ( const ProjectFile &str ) const
```

Comment:
can we rename "str" to "other" ? it is common naming scheme for operators and it is easier to follow the code then...